aroma/features.py

```diff
@@ -154,7 +153,7 @@ def feature_frequency(mel_FT_mix, TR):
     return HFC
 
 
-def feature_spatial(mel_IC):
+def feature_spatial(mel_IC, masks):
```

Comment:
```suggestion
def feature_spatial(*, z_maps, csf_mask, edge_mask, brain_mask):
```
Using a dict here is dangerous - you are risking an obscure `KeyError`. Making the three masks keyword arguments makes Python check they are correctly provided by name.

The out_mask is just the negated brain mask, right?

Reply:
That's a good point. Thank you for the recommendation.

> The out_mask is just the negated brain mask, right?

Correct.